Tests/Functional/app/AppKernel.php

```diff
@@ -98,6 +98,7 @@ public function registerContainerConfiguration(LoaderInterface $loader)
     {
         $loader->load($this->rootConfig);
         $loader->load(function (ContainerBuilder $container) {
+            $container->setParameter('container.autowiring.strict_mode', true);
```

Comment:
Needed for the Symfony 3.4 build, otherwise Symfony would try to create services for argument type hints where we rely on param converters. This parameter opts in for the Symfony 4 behaviour.